### DISCUSSION TITLE
Help with editor theme selection color

### DISCUSSION THREAD

**Discussion by clicktodev** [zed-industries/zed#16212] [Feature Requests]
Can someone help me find the right property to change the selection highlight color?
https://github.com/hichemfantar/halcyon-zed/issues/1
PRs are welcome

**Comment by bajrangCoder** [comment]
Try changing this value:

```
search.match_background
```

**Reply by clicktodev** [reply]
Thanks it worked!
do you know what the setting is for this:
![image](https://github.com/user-attachments/assets/20f4fdc2-8fa0-43ba-a84e-cc792535031d)

because the blue background looks broken in my theme:
![image](https://github.com/user-attachments/assets/3b7ddaf9-6399-45e4-8c48-7b9844ed165d)

**Reply by bajrangCoder** [reply]
I'm not sure about this but try this one:

```
editor.subheader.background
```

**Reply by clicktodev** [reply]
It's not working, the text is still blending with the background

**Reply by clicktodev** [reply]
I also have this problem where the selection color is completely opaque in the hover tooltip, would appreciate some help with this

![image](https://github.com/user-attachments/assets/fb761ad3-681d-4f59-abbd-051e4d74d7cd)

**Reply by bajrangCoder** [reply]
> It's not working, the text is still blending with the background

I got it.
Use `info.background`

**Reply by bajrangCoder** [reply]
> I also have this problem where the selection color is completely opaque in the hover tooltip, would appreciate some help with this
> 
> ![image](https://private-user-images.githubusercontent.com/34947993/358937075-fb761ad3-681d-4f59-abbd-051e4d74d7cd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwMzA5MzIsIm5iZiI6MTcyNDAzMDYzMiwicGF0aCI6Ii8zNDk0Nzk5My8zNTg5MzcwNzUtZmI3NjFhZDMtNjgxZC00ZjU5LWFiYmQtMDUxZTRkNzRkN2NkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE5VDAxMjM1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ3MGIwMTBhNmQ0NTNjZmYwNGQ5MDJlNmI5NGE3NDQxOTU3ZGNkNTczM2M3NWFmN2ExNzAzMjVkMzlhZTI5ZWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8x8hraiTihNdk6eAxQN2l--kDNZ-SNf-2qUQ-QimZqo)


You will need to add elements to the `players` array, with each object containing the following properties:

```json
{
  "cursor": "color...",
  "selection": "selection color...",
  "background": "background color..."
}
```

Please note that the first element in the `players` array is used as the value for the local user.

**Reply by clicktodev** [reply]
Thanks for helping fix the theme! highly appreciate it 🚀
i'd appreciate your feedback if you use the theme and help me make it better!
